e2e/_suites/ingest-manager/README.md

```diff
@@ -45,10 +45,10 @@ This is an example of the optional configuration:
 
    ```shell
    # There should be a Docker image for the runtime dependencies (elasticsearch, kibana, package registry)
-   export STACK_VERSION=7.9.0
+   export STACK_VERSION=7.9.1
    # (Fleet mode) This environment variable will use a fixed version of the Elastic agent binary, obtained from
-   # https://artifacts-api.elastic.co/v1/search/7.9.0/elastic-agent
-   export ELASTIC_AGENT_DOWNLOAD_URL="https://snapshots.elastic.co/7.9.0-81c43e7b/downloads/beats/elastic-agent/elastic-agent-7.9.0-linux-x86_64.tar.gz"
+   # https://artifacts-api.elastic.co/v1/search/7.9.1/elastic-agent
+   export ELASTIC_AGENT_DOWNLOAD_URL="https://staging.elastic.co/7.9.1-7b774025/downloads/beats/elastic-agent/elastic-agent-7.9.1-linux-x86_64.tar.gz"
```

Comment:
As 7.9.1. is out, I wonder why we use staging here? Couldn't we just use the stable release for the 7.9 testing?

Or if we want to test the snapshot, then it should probably be something like `docker.elastic.co/beats/elastic-agent:7.9.2-SNAPSHOT`? But this is not avaiable under artifacts yet (perhaps something to be changed?)

Reply:
If we query `https://artifacts-api.elastic.co/v1/search/7.9.1/elastic-agent` it returns that `staging` URL, and that has recently changed.

Related to 7.9.2, I'd ask: what versions do we want to test in the 7.9.x maintenance branch?

Reply:
from my perspective... I'd say, we kinda rather never plan to run a test against a GA version.  As funny as that may seem.  @ruflin - we'd always want to set the tests to pick up the latest code available from a given release line for all components (agent, kibana, es, relating-registry if applicable), the snapshots are that right?   

The question of what artifact registry is beyond me... things change and we will keep up with 'current'.  I was concerned at first that the 7.9.0 artifact was 'gone'... but if this is just an internal access point I guess that's ok and fine.

Syncing back to your question @mdelapenya - do we not have a way to dynamically leave the 7.9.x branch to pull the latest artifacts / snapshots available for 7.9.x instead of hard-coding it to a specific staging hash?  Let me look this up myself...